cmd/yurt-manager/app/manager.go

```diff
@@ -175,6 +177,14 @@ func Run(c *config.CompletedConfig, stopCh <-chan struct{}) error {
 		metricsServerOpts.ExtraHandlers[path] = handler
 	}
 
+	trim := func(obj interface{}) (interface{}, error) {
```

Comment:
how about rename function name `trim` to `trimManagedFields`?

Reply:
that's more straightforward. done